### PR TITLE
 Atlassian Beacon data connector solution

### PR DESCRIPTION
**Pull request by ltipp147**
Required items, please complete
   
   Change(s):
   - Atlassian Beacon solution: Analytic Rule (1); Logic App (1); Data Connector (1)
   - Solution has been packaged
   - Logo is added
   - Sample data is added

   Reason for Change(s):
   - New Sentinel Content Hub Solution as a Service - Atlassian Beacon

   Version Updated:
   - N/A

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by v-prasadboke** [comment]
Hello @ltipp147, Thanks for raising this PR. 
This PR will be investigated and will update you about the same before 29 September, 2023.

File: Solutions/Atlassian Beacon/Playbooks/Sync Alerts/azuredeploy.json
Comment (v-prasadboke) [inline Solutions/Atlassian Beacon/Playbooks/Sync Alerts/azuredeploy.json:1]: Also please add metadata to playbook. Metadata is missing from playbook. 
For reference please go through this playbook :- https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ThreatXCloud/Playbooks/ThreatXPlaybooks/ThreatX-BlockIP-URL/azuredeploy.json

**Comment by v-prasadboke** [comment]
Hello @ltipp147, Thanks for raising this PR. Please work on the above requested changes.

**Comment by ltipp147** [comment]
Hi @v-prasadboke, thank you for your prompt response. We will work on the request and keep you updated. 

In the meantime, I've encountered an issue with the LogoValidation check. The error message indicates that 'Id should be GUID format and uniquely identifiable'. Could you please provide some guidance on how to resolve this, please?

**Comment by v-prasadboke** [comment]
I've updated the GUID for logo

**Comment by ltipp147** [comment]
Hi @v-prasadboke, thank you for your update. I have completed the request, including the following:

1. Added and updated a new Logo file
2. Added Metadata section to the Playbook
3. Repackaged the solution with the latest update
4. Added ReleaseNotes.md

Any further requirement please let me know, thank you!

**Comment by v-prasadboke** [comment]
Hello @ltipp147, Thank you committing the requested changes. Will review this PR and update you about the same before 04 October, 2023.

**Comment by ltipp147** [comment]
Hi @v-prasadboke, any update on the PR please?

**Comment by v-atulyadav** [comment]
Hi @ltipp147,
We apologize for the delay, but due to lack of availability, we will update you on this by 06 Oct 2023. Thanks

File: Solutions/Atlassian Beacon/Playbooks/Sync Alerts/azuredeploy.json
Comment (v-prasadboke) [reply Solutions/Atlassian Beacon/Playbooks/Sync Alerts/azuredeploy.json:1]: Please add lastudpated time, post deployment, entities, tags, support

**Comment by v-prasadboke** [comment]
Also If required please add readme file for playbook which contains instructions/guidelines regarding playbook

**Comment by dark-binary** [comment]
Hey @v-prasadboke, the alert source by itself does not provide the tactics and technique in the alerts. For our last solution (Cortex XDR) publishment, I remember us agreeing that the tactics and technique could be left blank for the moment (since the source portal has its limitations in sending them in the right format). Is that still possible or are those fields now mandatory?

On a worst case we can add a generic tactic and technique, but would prefer waiting until the alert provider gives the capability to add tactics and techniques for the appropriate alert.

Please let us know what's best feasible and we can do it accordingly.

**Comment by v-prasadboke** [comment]
Hello @ltipp147, Yeah I Understand we skipped techniques and tactics in one of the Solution. It was just if the author has missed adding the entities and its our regular checklist to look for technique and tactics. 
Thanks, I'll continue with my investigation.

**Comment by dark-binary** [comment]
Hey @v-prasadboke, sure do. Does 9AM IST (4:30pm NZ time) work for you on 11/10/2023 or 12/10/2023?

**Comment by v-prasadboke** [comment]
Will 10:00 AM IST be okay for you for 12 October, 2023.

**Comment by ltipp147** [comment]
Hi @v-prasadboke, thank you for your time during our call. We have made the necessary updates to the Playbook metadata and repackaged the solution as per your request. Please review and let us know if you have any questions/further requirements.

Thanks!

**Review by v-prasadboke — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**